### ISSUE TITLE
Export options information

### ISSUE THREAD

**Issue by anne-pacalet** [dbuenzli/cmdliner#186]
In [another ticket](https://github.com/dbuenzli/cmdliner/issues/1), you said:

> Today while washing the dishes I thought maybe the best way to try to move on this is that when cmdliner is invoked with `--cmdliner` it should simply dump on stdout all the static information it knows about command lines in some form of structured data format like json or sexp.

For documentation purpose, I would love to be able to export everything about the options in a JSON file. Do you still have this plan in mind?

**Comment by dbuenzli** [comment]
Yes, the recent discussion [here](https://github.com/dbuenzli/cmdliner/pull/177)

**Comment by anne-pacalet** [comment]
Excellent! :+1:
